recipes/pysmb/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pysmb" %}
+{% set version = "1.2.1" %}
+{% set sha256 = "ab721bbdcf27a4a8a77e2f82dfc6d5e33b4db88f5075160a8bc065722a1270be" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pyasn1
+
+test:
+  imports:
+    - smb
+    - nmb
```

Can we try to add ``pip check`` here please?
```suggestion
  requires:
    - pip
  commands:
    - pip check
```